packages/gasket-plugin-intl/test/build-modules.test.js

```diff
@@ -0,0 +1,235 @@
+        fakeTgtDir = '/path/to/tgt/';
+
+        sinon.stub(builder, 'copyFile');
+        sinon.stub(builder, 'copyFolder');
+      });
+      it('calls copyFile for a json file', function () {
+        const fakeFileName = 'aa-AA.json';
+        const fakeFiles = [fakeFileName];
+
+        builder.processFiles(fakeSrcDir, fakeTgtDir, fakeFiles);
+
+        assume(builder.copyFile.getCalls()).length(1);
+        assume(builder.copyFolder.getCalls()).length(0);
+      });
+      it('calls copyFolder when its a folder', async function () {
+        const fakeFileName = 'some-dir';
+        const fakeFiles = [fakeFileName];
+
+        sinon.stub(fs, 'lstat').callsFake(() => {
+          return new Promise((resolve) => {
+            resolve({
+              isDirectory: () => {
+                return true;
+              }
+            });
+          });
+        });
+
+        await builder.processFiles(fakeSrcDir, fakeTgtDir, fakeFiles);
+
+        assume(builder.copyFile.getCalls()).length(0);
+        assume(builder.copyFolder.getCalls()).length(1);
+      });
+    });
+
+    describe('#getPackageNameFromDir', function () {
+      it('parses and returns the module name', function () {
+        assume(builder.getPackageNameFromDir('/src/path/to/test-app/locales')).equals('test-app');
+      });
+      it('parses and returns the scoped module name', function () {
+        assume(builder.getPackageNameFromDir('/src/path/to/@module/test-app/locales')).equals('@module/test-app');
+      });
+    });
+
+    describe('#getPackageName', function () {
+      it('parses and returns the app name', async function () {
+        sinon.stub(fs, 'readJson').resolves({ name: 'test-app' });
+        assume(await builder.getPackageName('/src/path/to/test-app/locales')).equals('test-app');
+      });
+      it('parses and returns the scoped module name', async function () {
+        sinon.stub(fs, 'readJson').resolves({ name: '@module/test-app' });
+        assume(await builder.getPackageName('/src/path/to/@module/test-app/locales')).equals('@module/test-app');
+      });
+      it('falls back to name from dir if no name in package.json', async function () {
+        sinon.stub(fs, 'readJson').resolves({ missing: 'name' });
+        assume(await builder.getPackageName('/src/path/to/@another/module/locales')).equals('@another/module');
+      });
+      it('falls back to name from dir if error reading package.json', async function () {
+        sinon.stub(fs, 'readJson').rejects('bad');
+        assume(await builder.getPackageName('/src/path/to/@another/module/locales')).equals('@another/module');
+      });
+    });
+
+    describe('#processDirs', function () {
+      let buildDirs;
+      beforeEach(function () {
+        buildDirs = [
+          '/src/test1/locales',
+          '/src/test2/locales'
+        ];
+        sinon.stub(fs, 'remove').returns(Promise.resolve());
+        sinon.stub(fs, 'mkdirp').returns(Promise.resolve());
```

Comment:
why not use `resolves`?
```suggestion
        sinon.stub(fs, 'remove').resolves();
        sinon.stub(fs, 'mkdirp').resolves();
```